src/test/java/org/fundacionjala/sfdc/stepdefinition/OpportunitySteps.java

```diff
@@ -0,0 +1,114 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.commons.DriverManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)